examples/with-passwordless/api-server.js

```diff
@@ -0,0 +1,559 @@
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const helmet = require("helmet");
+let supertokens = require("supertokens-node");
+let Session = require("supertokens-node/recipe/session");
+let { verifySession } = require("supertokens-node/recipe/session/framework/express");
+let { middleware, errorHandler } = require("supertokens-node/framework/express");
+let Passwordless = require("supertokens-node/recipe/passwordless");
+// let Twilio = require("twilio");
+let nodeMailer = require("nodemailer");
+let axios = require("axios").default;
+
+let mailTransporter = nodeMailer.createTransport({
+    host: "smtp.gmail.com",
+    port: 465,
+    secure: true,
+    auth: {
+        user: process.env.NODEMAILER_USER,
+        pass: process.env.NODEMAILER_PASSWORD,
+    },
+});
+
+const apiPort = process.env.REACT_APP_API_PORT || 3001;
+const apiDomain = process.env.REACT_APP_API_URL || `http://localhost:${apiPort}`;
+const websitePort = process.env.REACT_APP_WEBSITE_PORT || 3000;
+const websiteDomain = process.env.REACT_APP_WEBSITE_URL || `http://localhost:${websitePort}`;
+
+function getOTPBody(appName, codeLifetime, userInputCode) {
```

Comment:
Put all the html stuff into a different file. Make api-server a folder, and put this file in api-server/index.js. Rememeber to update the readme and other places to point to this new path